eth/filters/api.go

```diff
@@ -502,11 +501,11 @@ func (args *FilterCriteria) UnmarshalJSON(data []byte) error {
 		args.BlockHash = raw.BlockHash
 	} else {
 		if raw.FromBlock != nil {
-			args.FromBlock = big.NewInt(raw.FromBlock.Int64())
+			args.FromBlock = rpc.NewRPCBlockNumber(int(raw.FromBlock.Int64()))
```

Comment:
Aren't `raw.FromBlock` and `args.FromBlock` of the same type, can you just do assignment?

Reply:
Not really, golang doesn't allow `&` operation for type conversion but we have to convert int64 to int or rpc.BlockNumber.